app/assets/stylesheets/application.scss

```diff
@@ -78,6 +78,17 @@ footer {
   max-width: 500px;
 }
 
+.tattoo-box {
+  background-color: rgb(32, 32, 32);
+  margin: auto;
+  text-align: center;
+}
+
+.fillwidth {
+    min-width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, min-width